src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -214,6 +214,18 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                     _success = false;
                 }
 
+                if (_request.Project?.RestoreMetadata != null)
+                {
+                    foreach (var source in _request.Project.RestoreMetadata.Sources)
```

Comment:
does this collection exclude disabled sources?

Reply:
nope, only enabled.